label_sync/labels.yaml

```diff
@@ -128,12 +128,15 @@ labels:
     name: sig/contributor-experience
   - color: d2b48c
     name: sig/docs
-  - color: d2b48c
-    name: sig/federation
   - color: d2b48c
     name: sig/gcp
   - color: d2b48c
     name: sig/instrumentation
+  - color: d2b48c
+    name: sig/multicluster
+    previously:
+      - name: sig/federation
```

Comment:
Remove this? Not sure why is the name listed twice.

Reply:
Looking at k/k _both_ of these labels show up right now, so this is intended to catch both
- https://github.com/kubernetes/kubernetes/labels/sig%2Ffederation%20%28deprecated%20-%20do%20not%20use%29
- https://github.com/kubernetes/kubernetes/labels/sig%2Ffederation

We can remove one for the other later